linera-chain/src/chain.rs

```diff
@@ -147,8 +148,9 @@ static STATE_HASH_COMPUTATION_LATENCY: LazyLock<HistogramVec> = LazyLock::new(||
     )
 });
 
-/// The BCS-serialized size of an empty `ExecutedBlock`.
-const EMPTY_EXECUTED_BLOCK_SIZE: usize = 91;
+/// The BCS-serialized size of an empty [`Block`].
+// TODO: UPDATE
+const EMPTY_BLOCK_SIZE: usize = 91;
```

Comment:
TODO: Update new empty block size.

Reply:
Done in [Testy empty Block size.](https://github.com/linera-io/linera-protocol/pull/3101/commits/5fe042587db04596109fa57d5d66cb29303122bc)